packages/accounts-password/password_server.js

```diff
@@ -500,11 +500,16 @@ Accounts.setPassword = function (userId, newPlaintextPassword, options) {
 Meteor.methods({forgotPassword: function (options) {
   check(options, {email: String});
 
-  var user = Meteor.users.findOne({"emails.address": options.email});
+  var user = Accounts._findUserByQuery({email: options.email});
   if (!user)
     throw new Meteor.Error(403, "User not found");
 
-  Accounts.sendResetPasswordEmail(user._id, options.email);
+  const emails = _.pluck(user.emails || [], 'address');
+  const caseSensitiveEmail = _.find(emails, email => {
```

Comment:
The case I had in mind isn't hard at all to just pick the stored email and use that, but just over all. If the account system support case insensitive emails I would expect that to be the case for all email related functions, or is my thinking completely flawed? :stuck_out_tongue_closed_eyes: